ql/src/test/queries/clientnegative/except_cbooff.q

```diff
@@ -0,0 +1,5 @@
+set hive.cbo.enable=false;
```

Comment:
Do we have positive tests?

Reply:
Yes there are, they were added when the original feature was developed.